Gemfile

```diff
@@ -1,7 +1,6 @@
 source 'https://rubygems.org'
 
-# Make sure to bump .ruby-version when updating this
-ruby '2.6.2'
+ruby File.read('.ruby-version').strip
```

Comment:
i. love.this. so. much.

Reply:
Ha, I'm a `chruby` user as well. :wink:

Reply:
I dig it.  When I catch up on things mid week, I'm going to see if we can also have our dockerfile reference .ruby-version as well, since that's the other place that we've struggled to keep in sync.

Reply:
I've seen Docker images built that way before but don't *exactly* remember how to do that. 😅

Circle CI, for this project, could use dynamic Ruby version support as well. [Here is the solution](https://github.com/bkuhlmann/gemsmith/blob/master/circle.yml#L16-L42) I've ended up using for my open source projects. I could update Circle CI for this project too if you like? The caveat is that it does require a full Ruby build which *does slow down the builds* but like *not* having to maintain that file either.